src/pages/Authentication/Login/Login.jsx

```diff
@@ -0,0 +1,115 @@
+import { useState } from "react";
+import { Navbar } from "../../../components/Navbar/Navbar";
+import { Footer } from "../../../components/Footer/Footer";
+import "../styles.css";
+import { useAuth } from "../../../contexts/authContext";
+import { loginService } from "../../../services/loginService";
+import { Link } from "react-router-dom";
+
+const Login = () => {
+  const [loginInput, setLoginInput] = useState({});
+  const [hide, setHide] = useState({ pwd: true });
+
+  const { setIsAuth, setToken } = useAuth();
+
+  const loginInputHandler = (e) => {
+    const { name, value } = e.target;
+    setLoginInput({ ...loginInput, [name]: value });
+  };
+
+  const loginHandler = async (e) => {
+    e.preventDefault();
+
+    try {
+      const { data } = await loginService(loginInput);
+
+      localStorage.setItem("token", data.encodedToken);
+      setToken(data.encodedToken);
+
```

Comment:
You can redirect the user to a different page, when the user logs in.
```js
// Just an example, you can try - 
 const navigate = useNavigate()
onClick={() => navigate('your-route')}
```

Reply:
Yes, I will implement this. Thanks for elaborating on the example.